mkdocs.yml

```diff
@@ -0,0 +1,85 @@
+site_name: Orchard Core Commerce
+theme:
+  name: material
+  custom_dir: src/docs/theme
```

Comment:
Can we make these directory names inside `src/` capitalized?
To be consistent with the others like Modules and Libraries.

Reply:
Sorry, I won't change this for the moment but I am thinking about moving the src\docs to the root:
- It is only placed there so we can make links to code file (which we rarely do).
- It would solve your problem with the duplicated Readme (at root and in src).

Reply:
Moving docs to root sounds good.

Reply:
docs moved to root.